tests/test-cases/Group3-Docker-Compose/3-03-Docker-Compose-Basic.robot

```diff
@@ -61,6 +71,16 @@ Compose kill
     Log  ${out}
     Should Be Equal As Integers  ${rc}  0
 
+    ${rc}  ${out}=  Run And Return Rc And Output  docker %{VCH-PARAMS} ps -a
+    Log  ${out}
+    Should Be Equal As Integers  ${rc}  0
+
+    ${rc}  ${out}=  Run And Return Rc And Output  docker %{VCH-PARAMS} network inspect vic_default
+    Log  ${out}
+    Should Be Equal As Integers  ${rc}  0
+
+    Wait Until Keyword Succeeds  1 min  5 sec  Check Container Removed From Network  vic_redis_1  vic_default
```

Comment:
Ideally it should be done within seconds since we run it on ESXi instead of VC. I could bump it up to 5 min. If it doesn't succeed within 5 min, definitely we have an issue here. What do you think?

Even if we run this test on VC, the host sync for power state update between the vcenter server and the esxi host is around 20 sec. So if the network is ok, we should be able to receive this event within 1 min.

Reply:
Sounds good. A "correct" timeout is difficult to determine here, since I'm not aware of an upper bound to the event time delay. I was thinking 2 mins, but if you think 1 min is sufficient, that's fine too.